homeassistant/components/vacuum/neato.py

```diff
@@ -82,6 +83,7 @@ def update(self):
             self._available = False
             return
         _LOGGER.debug('self._state=%s', self._state)
+        self._robot_alert = ALERTS.get(self._state['alert'])
```

Comment:
If we only use the variable locally in this method, the variable can be local and not an instance attribute.

Reply:
Thank you for the review :) I have now updated this variable to be local.